### PR TITLE
LSTM data loaders for single step prediction using sequential and random batch generators

### PR DESCRIPTION
**Pull request by Saran-nns**
Warmsteps are removed intentionally. TODO: Use the warmup steps feature in Optimizers.

### CODE REVIEW

File: traja/models/nn.py [hunk 1]
```diff
@@ -122,10 +240,11 @@ def __init__(self, model,
...
         self.train_loader = train_loader
         self.test_loader = test_loader
-
         self.warmup_steps = warmup_steps
-
-        self.criterion = LossMseWarmup(self.warmup_steps)
+        if self.warmup_steps!=None: 
+            self.criterion = LossMseWarmup(self.warmup_steps)
+        else:
+            self.criterion = LossMse(self.warmup_steps) # warmup_steps == None
```
Comment (WolfByttner) [inline traja/models/nn.py:247]: Why accept warmup steps if we don't use them?

Comment (Saran-nns) [reply traja/models/nn.py:247]: It is just a check to have backward compatibility with other models. There are few models in  nn.py which are still not tested. Currently, we are using only LSTMs. But yes, I also strongly discourage warmup steps. We could use warm steps feature inside the optimizer. For  that, I will create an issue instead and assign myself to address it.

Comment (Saran-nns) [reply traja/models/nn.py:247]: #24  is the issue

Comment (Saran-nns) [reply traja/models/nn.py]: Added user-defined trial dir name, removed the warmup steps and correponding loss functions. In the future, a warmup scheduler will be added.

**Review by WolfByttner — APPROVED** [review]

**Review by justinshenk — COMMENTED** [review]
Is this final or should I disregard?

**Comment by Saran-nns** [comment]
Files in this PR will be overridden in the new PR. The new PR will have Stacked LSTM layers and Autoencoders for step and multi-step predictions.
